dts/msm8916/msm8916-mtp.dts

```diff
@@ -3,9 +3,12 @@
 /dts-v1/;
 
 #include <skeleton.dtsi>
+#include <lk2nd.h>
 
 / {
 	compatible = "qcom,msm8916-mtp", "qcom,msm8916";
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <8 0>;
```

Comment:
It doesn't look right to change this file, if it's not an 1gb modem (`qcom,board-id = <8 0>;`).
Check your downstream device tree to confirm.